meilisearch-release-changelog/v1.6.0.md

```diff
@@ -0,0 +1,202 @@
+# v1.6.0 release changelogs
+
+<!-- The following line should ONLY be put PRE-release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+Meilisearch v1.6 focuses on improving indexing performance. This new release also adds hybrid search and simplifies the process of generating embeddings for semantic search.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+Some SDKs might not include all new features—consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and improvements 🔥
+
+## Experimental: Automated embeddings generation for vector search
+
+With v1.6, you can configure Meilisearch so it automatically generates embeddings using either OpenAI or HuggingFace. If neither of these third-party options suit your application, you may provide your own embeddings manually:
+
+- `openAI`: Meilisearch uses the OpenAI API to auto-embed your documents. You must supply an OpenAPI key to use this embedder
+- `huggingFace`: Meilisearch automatically downloads the specified `model` from HuggingFace and generates embeddings locally. This will use your CPU and may impact indexing performance
+- `userProvided`: Compute embeddings manually and supply document vectors to Meilisearch. You may be familiar with this approach if you have used vector search in a previous Meilisearch release. Read further for details on breaking changes for user provided embeddings usage
+
+### Usage
+
+Use the `embedders` index setting to configure embedders. You may set multiple embedders for an index. This example defines 3 embedders named `default`, `image` and `translation`:
+
+```sh
+curl \
+  -X PATCH 'http://localhost:7700/indexes/movies/settings' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "embedders": {
+      "default": {
+        "source":  "openAi",
+        "apiKey": "<your-OpenAI-API-key>",
+        "model": "text-embedding-ada-002",
+        "documentTemplate": "A movie titled '{{doc.title}}' whose description starts with {{doc.overview|truncatewords: 20}}"
```

Comment:
Will this code sample actually work with the nested single quotes `'` we're feeding `--data-binary`? Should we escape `'{{doc.title}}'`?

Reply:
Hum you are right, we need to escape them!

Reply:
Done!